src/test/scala/viper/gobra/GobraParserTests.scala

```diff
@@ -0,0 +1,89 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+
+package viper.gobra
+
+import ch.qos.logback.classic.Level
+import org.scalatest.BeforeAndAfterAll
+import viper.gobra.frontend.Source.FromFileSource
+import viper.gobra.frontend.{Config, PackageResolver}
+import viper.gobra.reporting.VerifierResult.{Failure, Success}
+import viper.gobra.reporting.{NoopReporter, VerifierError}
+import viper.gobra.util.{DefaultGobraExecutionContext, GobraExecutionContext}
+import viper.silver.testing.{AbstractOutput, AnnotatedTestInput, ProjectInfo, SystemUnderTest}
+import viper.silver.utility.TimingUtils
+
+import java.nio.file.Path
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class GobraParserTests extends AbstractGobraTests with BeforeAndAfterAll {
+
+  val regressionsPropertyName = "GOBRATESTS_REGRESSIONS_DIR"
+
+  val regressionsDir: String = System.getProperty(regressionsPropertyName, "better_errors")
+  val testDirectories: Seq[String] = Vector(regressionsDir)
```

Comment:
You can directly use the specified directory without the option to allow users to customize it

```suggestion
 val testDirectories: Seq[String] = Vector("better_errors")
```

Reply:
What's about the `unimplemented_go` folder? What kind of test cases are stored in there?

Reply:
They include things that the parser can parse, but either don't have a representation in the Gobra AST or fail at some later stage.